internal/pkg/manifest/backend_svc_test.go

```diff
@@ -905,3 +905,41 @@ func TestBackendSvc_ApplyEnv_CountOverrides(t *testing.T) {
 		})
 	}
 }
+
+func TestBackendSvc_IsServiceAvailableInRegion(t *testing.T) {
+	testCases := map[string]struct {
+		in      *BackendService
+		region  string
+		want    bool
+		wantErr error
+	}{
+		"region exist": {
+			region:  "us-west-1",
+			want:    true,
+			wantErr: nil,
```

Comment:
nit: probably redundant since `wantErr` is initialized as `nil`.
```suggestion
```

Reply:
It's actually not clear what these unit tests in `manifest` package are testing 🤔 These tests seems to be testing `regions.IsServiceAvailable`, instead of `manifest`'s `IsServiceAvailable`

Reply:
Yeah right, first I thought it is important to have a unit test for each kind of a service for example loadbalanced service or backend service and etc. But then I realized it is enough to write a test for `regions.IsServiceAvailable` with only two scenarios i.e. ecs service and apprunner service. So I have removed tests from other `service_test` files and have only one test for `regions.IsServiceAvailable`